### PR TITLE
adding Targets

### PR DESCRIPTION
**Pull request by Kaiowarez**
adding targets

HGLRCF411 - TESTED GOOD
IFLIGHT F411 PRO - TESTED GOOD
IFLIGHT F4 AIO - TESTER NEEDED
IFLIGHT F4 TWIN G - TESTER NEEDED
IFLIGHT F7 TWIN G - TEST IN PROGRESS SO FAR SO GOOD
LUXF7MINI - TESTER STILL TESTING
MAMBAF405US - TESTER NEEDED
MAMBAF411 - TESTER NEEDED
NBDINIFINITY - RENAMED
OMNIBUSF4NANOV7 - TESTER NEEDED
OMNIBUSF7NANOV7 - TESTER NEEDED
SPEEDYBEEF7 - TESTER NEEDED
TALONF7FUSION -  FROM HENA C/O MADCHILLER
TMOTORF7 - TESTED UNCONFIRMED
ZEEZF7 - TESTING IN PROGRESS

ANY OF THE TARGETS YOU WISH TO TEST PERSONALLY DM ME IN DISCORD THANKS

### CODE REVIEW

File: src/main/target/A_TEMPLATE/target.mk.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+F405_TARGETS    += $(TARGET)
```
Comment (gretel) [inline src/main/target/A_TEMPLATE/target.mk.md]: Codacy Issue found: [Could not find module `remark-lint-no-unneeded-full-reference-image`](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5221948)

**Comment by gretel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5221948&bid=17121466)

**Comment by Kaiowarez** [comment]
Ah that A_TEMPLATE FOLDER is like my reference template i keep forgetting
to not add it..

**Comment by gretel** [comment]
> Ah that A_TEMPLATE FOLDER is like my reference template i keep forgetting to not add it..

thanks! please add the folder to your `.gitignore` file.

please remote the folder and fix the conflict so it gets ready for merge.

**Comment by gretel** [comment]
@Kaiowarez please try to tidy one of the PRs up and close the other and do not open multiple PRs for the same progress. thank you!

**Comment by Kaiowarez** [comment]
Will do

**Pull request closed without merging**